From 04ae7b07eef0a92f7d62bcb53ec024bf4b861544 Mon Sep 17 00:00:00 2001
From: Tianshu Qiu <tian.shu.qiu@intel.com>
Date: Thu, 26 Mar 2015 18:36:09 +0800
Subject: [PATCH 326/338] atomisp2: Fix Makefile to support ISP2300

Change-Id: Ica76d7d6650149212926f48471301217f6e9dcb2
Fixes-Issue: #H1503933662
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/48447
Validation-Android: Wei Tao <wei.tao@intel.com>
Approver: Bin B Han <bin.b.han@intel.com>
Reviewed-by: Bin B Han <bin.b.han@intel.com>
Build: IMIN-legacy (SC)
Integrator: Wei Tao <wei.tao@intel.com>
Build: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
Maintainer: Wei Tao <wei.tao@intel.com>
---
 .../external_drivers/camera/drivers/media/pci/atomisp2/Makefile.common  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/Makefile.common b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/Makefile.common
index 860732a..ca02407 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/Makefile.common
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/Makefile.common
@@ -71,7 +71,7 @@ DEFINES += -DSYSTEM_hive_isp_css_2401_system -DISP2401
 else ifneq ($(ISP2400B0), )
 DEFINES += -DSYSTEM_hive_isp_css_2400_system -DISP2400
 else
-$(error Undefined HIVE ISP CSS system version)
+DEFINES += -DSYSTEM_hive_isp_css_system
 endif
 
 ccflags-y += $(INCLUDES) $(DEFINES) -fno-common -Werror
-- 
1.9.1

